config/redis.yml

```diff
@@ -14,6 +14,9 @@ development: &defaults
   user_store:
     namespace: users
     each_ttl: 3600
+  user_identity_store:
+    namespace: user_identities
```

Comment:
The namespace convention in this file seems to be kebab-case rather than snake_case.